pkg/cmd/grafana-cli/main.go

```diff
@@ -16,7 +17,7 @@ var version = "master"
 
 func main() {
 	setupLogging()
-
+	flag.Parse()
```

Comment:
this makes it possible to override settings by passing args to the command line. ex
```bash
/usr/sbin/grafana-cli admin data-migration encrypt-datasource-passwords \
  --homepath=/usr/share/grafana \
  --config=/etc/grafana/grafana.ini \
  cfg:default.paths.data="$DATA_DIR" \
  cfg:default.paths.logs="$LOG_DIR"
```

Reply:
Interesting! In my branch, I took the flags from the context as they're parsed by the CLI. Within the `dbCommandFlags` function.